src/store/getters.ts

```diff
@@ -209,15 +209,5 @@ export default {
         EnumUtilities.isTypeRestorationLimitedExtension(stateFiling)
       )
     )
-  },
-
-  /**
-   * True if the business is authorized to continue out, ie, the state filing is a
-   * Consent to Continuation Out filing.
-   */
-  isAuthorizedToContinueOut (_state: StateIF, getters: any): boolean {
```

Comment:
I was thinking to maybe leave this getter here and implement the logic of the getter above here. @severinbeauvais @ketaki-deodhar thoughts?

Reply:
I have no strong opinion, but it does make sense to have it in the FHL store since that's where the filings array is.

Reply:
Leaning towards keeping in filing history store

Reply:
Makes sense! I'm resolving this conversation 👍

Reply:
All, please never stop asking questions. This is great! And we all learn some technical stuff and some wisdom 😃